app/javascript/services/forest-data.js

```diff
@@ -98,3 +107,28 @@ export const getFAOExtent = ({ country, period }) => {
     .replace('{period}', period);
   return axios.get(url);
 };
+
+export const getGainRanking = ({
```

Comment:
Again, unsure if this is needed. I have done something similar in for the FAO reforestation widget which now only uses one query and uses just selectors. Take a look and let me know your thoughts.